src/Application.cpp

```diff
@@ -473,6 +474,68 @@ void Application::initPubSub()
                 });
             });
 
+    std::ignore =
+        this->twitch->pubsub->signals_.moderation.suspiciousMessageReceived
+            .connect([&](const auto &action) {
+                if (action.treatment !=
+                    PubSubLowTrustUsersMessage::Treatment::Restricted)
+                {
+                    return;
```

Comment:
Maybe add a comment here stating that we currently only show restricted messages, but that we have plans to highlight monitored users soon(tm)